pkg/server/chaosd/redis.go

```diff
@@ -52,6 +53,22 @@ func (redisAttack) Attack(options core.AttackConfig, env Environment) error {
 
 	case core.RedisSentinelStopAction:
 		return env.Chaos.shutdownSentinelServer(attack, cli)
+
+	case core.RedisCacheLimitAction:
+		cacheSize, err := cli.ConfigGet(cli.Context(), "maxmemory").Result()
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		// Get the value of maxmemory
+		attack.OriginCacheSize = fmt.Sprint(cacheSize[1])
```

Comment:
seems that `cacheSize` is an array,  what is the `cacheSize[0]`?

Reply:
Because `redis.SetConfigSet().Result()` returns an interface list like `[maxmemory 100MB]`, the second one is the value what we needed.

Reply:
OK, it's better to add a comment for it, add the example of the cacheSize

Reply:
Updated